lib/jekyll/inclusion.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Jekyll
+  class Inclusion
+    attr_reader :name, :path
+
+    def initialize(site, base, name)
+      @site = site
+      @name = name
+      @path = PathManager.join(base, name)
+    end
+
+    def relative_path
```

Comment:
I don't see where this is used outside this class and suggest that we make it private.